src/encoded/tests/data/inserts/experiment.json

```diff
@@ -52,7 +52,8 @@
         "dbxrefs": [
             "UCSC-ENCODE-mm9:wgEncodeEM002001",
             "GEO:GSM798326",
-            "GEO:GSE32218"
+            "GEO:GSE32218",
```

Comment:
Do you to also update the unit and BDD experiment search pages?

Reply:
I updated the batch download test. All the other tests passed. I think it's ok, then?